.buildkite/nightly-benchmarks/kickoff-pipeline.sh

```diff
@@ -10,18 +10,24 @@ apt install -y curl jq
 # Install minijinja for templating
 curl -sSfL https://github.com/mitsuhiko/minijinja/releases/latest/download/minijinja-cli-installer.sh | sh
 source $HOME/.cargo/env
+target_yaml_file=""
 
 # If BUILDKITE_PULL_REQUEST != "false", then we check the PR labels using curl and jq
 if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then
   PR_LABELS=$(curl -s "https://api.github.com/repos/vllm-project/vllm/pulls/$BUILDKITE_PULL_REQUEST" | jq -r '.labels[].name')
 
   if [[ $PR_LABELS == *"perf-benchmarks"* ]]; then
-    echo "This PR has the 'perf-benchmarks' label. Proceeding with the nightly benchmarks."
-  else
-    echo "This PR does not have the 'perf-benchmarks' label. Skipping the nightly benchmarks."
-    exit 0
+    echo "This PR has the 'perf-benchmarks' label. Proceeding with the performance benchmarks."
+    target_yaml_file=".buildkite/nightly-benchmarks/benchmark-pipeline.yaml"
+  fi
+
+  if [[ $PR_LABELS == *"nightly-benchmarks"* ]]; then
```

Comment:
This means PR authors cannot test both in one PR?

Reply:
Fixed by uploading two pipelines when both labels are used

Reply:
I will add `perf-benchmarks` label to test if it works out as expected when adding 2 labels

Reply:
This file is actually inside `vllm-project/buildkite-ci` repo. I made a PR there. Can you check?